### PR TITLE
Broaden frontend software criteria

### PR DESCRIPTION
**Pull request by TechFanTheo**
Changes proposed in this PR:

Add that website mandating an account and blocking VPNs can also be included. We may also include website that block TOR, but that would significantly escape it's scope, as a lot of websites block TOR．

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### CODE REVIEW

**Comment by jonaharagon** [comment]
Thanks for the PR! However, we recommend frontends for which those criteria would not apply, so we can't list them. For example YouTube doesn't block access without an account, but we still recommend frontends for it.

**Comment by TechFanTheo** [comment]
I was thinking of an OR situation, but this wasn't clear. It should be at least one of the criteria. This is necessary because technically Old Reddit doesn't meet the criteria, while it blocks VPN which is absurd.

**Comment by dngray** [comment]
> while it blocks VPN which is absurd.

whereas old. tor onion works just fine.

File: docs/frontends.md [hunk 1]
```diff
@@ -255,3 +255,5 @@ Recommended frontends...
 We only consider frontends for websites which are...
 
 - Normally only accessible with JavaScript enabled.
+- Normally only accesible with an Account.
+- Block access from VPNs.
```
Comment (jonaharagon) [inline docs/frontends.md:259]:
```suggestion
We only consider frontends if one of the following is true for a platform:

- Normally only accessible with JavaScript enabled.
- Normally only accessible with an account.
- Blocks access from commercial [VPNs](vpn.md).
```

**Review by dngray — APPROVED** [review]

**Review by jonaharagon — APPROVED** [review]

**Pull request merged**